setup.py

```diff
@@ -27,7 +27,7 @@ def read(*filenames, **kwargs):
     author='Trevor Creech',
     version=about['__version__'],
     install_requires=[
-        'pep8',
+        'flake8',
```

Comment:
`flake8` has a dependency on (depending on version) either `pep8` or `pycodestyle`. (Note that they are the same project, [just renamed](https://github.com/PyCQA/pycodestyle/issues/466)). Since the code appears compatible with either, I think it makes sense to just let it be pulled in as a transitive dependency.